Documentation~/TableOfContents.md

```diff
@@ -0,0 +1,4 @@
+* [UPM Package Starter Kit](index)
+	* [Guide for features packages](tools-package-guide)
+	* [Guide for sample packages](sample-package-guide)
+	* [Guide for test packages](test-package-guide)
```

Comment:
@jasons-unity should we cut this or update it?

Reply:
Looks like this has already been addressed